style.css

```diff
@@ -0,0 +1,663 @@
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.active .bar:nth-child(1) {
+  transform: translateY(6px) rotate(45deg);
+  background-color: #fff;
+  position: fixed;
+  z-index: 1;
+  margin-top: 25px;
+}
+
+.active .bar:nth-child(3) {
+  transform: translateY(-6px) rotate(-45deg);
+  background-color: #fff;
+  position: fixed;
+  z-index: 1;
+  margin-top: 25px;
+}
+
+.cover {
+  background: url(Images/background.cov.jpg) no-repeat center center/cover;
+  padding: 0;
+  margin: 0;
+  padding-bottom: 10px;
+}
+
+.btn {
+  border: 2px solid #ec5242;
+  color: #ec5242;
+  display: none;
+}
+
+.first {
+  font-weight: 400;
+  padding: 0 10px;
+  color: #ec5242;
+}
+
+.second {
+  font-weight: 900;
+  color: #ec5242;
+  padding: 0 20px;
+  font-size: 25px;
+  margin: 0;
+}
+
+.second-pr {
+  margin: 0 20px;
+  font-weight: 400;
+  color: #272a31;
+  margin-bottom: 15px;
+}
+
+.dates {
+  font-weight: 700;
+  margin: 0 20px;
+}
+
+@media (min-width: 768px) {
+  header {
+    display: flex;
+    margin: 0 10px;
+    justify-content: space-between;
+    padding: 15px;
+  }
+
+  .main-page {
+    margin: 0;
+  }
+
+  .top-bg {
+    background: #272a31;
+  }
+
+  .fa-brands {
+    color: #fff;
+  }
+
+  .top {
+    display: flex;
+    color: #fff;
+  }
+
+  .lg-logo {
+    display: flex;
+    padding: 0 10px;
+  }
+
+  .logo {
+    padding: 5px;
+    font-weight: 700;
+    display: block;
+    text-decoration: none;
+    color: #272a31;
+    text-align: center;
+  }
+
+  .list-la {
+    color: #ec5242;
+    display: flex;
+    font-size: 15px;
+    font-weight: 300;
+    padding: 5px;
+    text-decoration: none;
+  }
+
+  .list {
+    display: flex;
+    gap: 20px;
+    list-style: none;
+    font-weight: 300;
+    color: #272a31;
+    padding: 5px;
+    align-items: center;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .go {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .btn {
+    border: 2px solid #ec5242;
+    color: #ec5242;
+    display: flex;
+    font-size: 18px;
+    font-weight: 500;
+    padding: 5px;
+    text-decoration: none;
+  }
+
+  .nav-1 {
+    gap: 5%;
+    padding: 0;
+    margin: 0;
+  }
+
+  .first {
+    font-weight: 400;
+    padding-top: 10px;
+    color: #ec5242;
+    margin: 0;
+  }
+
+  .second {
+    font-weight: 900;
+    color: #ec5242;
+    margin: 20px;
+    font-size: 25px;
+    margin-bottom: 15px;
+  }
+
+  .second-pr {
+    margin: 20px;
+    font-weight: 400;
+    color: #272a31;
+    margin-bottom: 15px;
+  }
+
+  .dates {
+    font-weight: 700;
+    margin: 20px;
+  }
+
+  .menu {
+    cursor: pointer;
+    align-self: center;
+    display: none;
+  }
+
+  .bar {
+    width: 15px;
+    margin: 4px auto;
+    background-color: black;
+    height: 2px;
+    display: none;
+  }
+}
+
+.card-section {
+  background-color: #272a31;
+  margin: 0;
+  padding: 20px 0;
+}
```

Comment:
The font sizes and dimensions used for the card section makes it difficult to read especially the paragraph/texts. :dark_sunglasses:  Kindly update it and make sure your design looks exactly as the given design guidelines to make it more readable and up to standard. Please ensure to use the same font-size, width, height and dimensions as the given design. This applies to all other parts of your project. :pray: 


 Your design            |  The actual design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-29 11-48-44](https://user-images.githubusercontent.com/65233860/204521185-d64fef2b-0750-4af7-87ec-e7806ad9d1f3.png)  |  ![Screenshot from 2022-11-29 11-28-12](https://user-images.githubusercontent.com/65233860/204521215-aa8929c5-6831-427b-96f4-46f0ab5c8522.png)


 Your design            |  The actual design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-29 11-51-38](https://user-images.githubusercontent.com/65233860/204521725-f25ff993-2279-481b-9b49-6c78a4f23db1.png)  |  ![Screenshot from 2022-11-29 11-27-58](https://user-images.githubusercontent.com/65233860/204521441-f76bb425-165d-44b2-819c-dea1fb573fec.png)